ui/page.go

```diff
@@ -420,38 +425,53 @@ func (page pageCommon) layoutAppBar(gtx layout.Context) layout.Dimensions {
 		gtx.Constraints.Min.X = gtx.Constraints.Max.X
 		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 			layout.Rigid(func(gtx C) D {
-				m := values.MarginPadding5
-				return layout.Inset{
-					Top:    m,
-					Bottom: m,
-					Left:   m,
-					Right:  values.MarginPadding15,
-				}.Layout(gtx, func(gtx C) D {
-					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
-						layout.Rigid(func(gtx C) D {
-							return layout.Inset{Top: values.MarginPadding9, Left: values.MarginPadding15}.Layout(gtx,
-								func(gtx layout.Context) layout.Dimensions {
-									img := page.icons.logo
-									img.Scale = 1.0
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								img := page.icons.logo
+								img.Scale = 0.6
+								return layout.Inset{
+									Top:   values.MarginPadding10,
+									Left:  values.MarginPadding25,
+									Right: values.TextSize16,
```

Comment:
Using textsize as a margin dimension doesn't look natural. The dimension should be defined as MarginPadding